ACM_General/events/models.py

```diff
@@ -47,6 +47,46 @@ def get_path_for_flier(instance, filename):
     )
 
 
+class Tag(models.Model):
+    """
+    Class used for adding classification tags on events.
+    """
+
+    #: The name of the tag that is displayed on the event creation form; represented
+    #: as a CharField.
+    display_name = models.CharField(
+        verbose_name=_('Event Tag'),
+        help_text=_('Different categories for the event.'),
+        max_length=256,
+    )
+
+    #: Unique id of the tag; represented as an IntegerField.
+    id = models.IntegerField(
```

Comment:
What I was thinking for this id was another `CharField` which has an unchanging name that the creator of this Tag creates. This makes things look better from the API perspective when requesting event data.